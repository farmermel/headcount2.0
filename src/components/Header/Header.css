header h1 {
  display: flex;
  justify-content: center;
  align-items: center;
  font-family: 'Roboto', sans-serif;
  height: 200px;
}