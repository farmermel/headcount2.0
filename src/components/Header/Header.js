import React from 'react';
import Search from "../Search/Search";
import './Header.css';

const Header = ({searchDistricts}) => {
  return (
    <header>
      <h1>HeadCount2.0</h1>
      <Search searchDistricts={searchDistricts} />
    </header>
  );
};

export default Header;