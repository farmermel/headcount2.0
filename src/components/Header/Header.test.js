import React from 'react';
import ReactDOM from 'react-dom';
import Header from './Header';
import { shallow } from 'enzyme';

describe('Header', () => {
  let wrapper;
  const mockSearch = jest.fn();

  it('should match the snapshot', () => {
    wrapper = shallow(<Header searchDistricts={mockSearch} />);
    expect(wrapper).toMatchSnapshot();
  });
});
