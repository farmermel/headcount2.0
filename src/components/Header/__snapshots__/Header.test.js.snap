// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should match the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Header
    searchDistricts={[Function]}
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <div
          className="header-wrap"
>
          <img
                    alt="check"
                    className="check-icon"
                    src="sign.svg"
          />
          <h1
                    className="title"
          >
                    HeadCount2.0
          </h1>
</div>,
        <Search
          searchDistricts={[Function]}
/>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <img
              alt="check"
              className="check-icon"
              src="sign.svg"
/>,
            <h1
              className="title"
>
              HeadCount2.0
</h1>,
          ],
          "className": "header-wrap",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "alt": "check",
              "className": "check-icon",
              "src": "sign.svg",
            },
            "ref": null,
            "rendered": null,
            "type": "img",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "HeadCount2.0",
              "className": "title",
            },
            "ref": null,
            "rendered": "HeadCount2.0",
            "type": "h1",
          },
        ],
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "searchDistricts": [Function],
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
    ],
    "type": "header",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="header-wrap"
>
            <img
                        alt="check"
                        className="check-icon"
                        src="sign.svg"
            />
            <h1
                        className="title"
            >
                        HeadCount2.0
            </h1>
</div>,
          <Search
            searchDistricts={[Function]}
/>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <img
                alt="check"
                className="check-icon"
                src="sign.svg"
/>,
              <h1
                className="title"
>
                HeadCount2.0
</h1>,
            ],
            "className": "header-wrap",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "alt": "check",
                "className": "check-icon",
                "src": "sign.svg",
              },
              "ref": null,
              "rendered": null,
              "type": "img",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "HeadCount2.0",
                "className": "title",
              },
              "ref": null,
              "rendered": "HeadCount2.0",
              "type": "h1",
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "searchDistricts": [Function],
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": "header",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
